openapi.yaml

```diff
@@ -209,9 +438,324 @@ definitions:
+        properties:
+          type:
+            $ref: '#/definitions/gameType'
+          attributes:
+            type: object
+            properties:
+              playerIds:
+                $ref: '#/definitions/playerIds'
+              minimumBet:
+                $ref: '#/definitions/minimumBet'
+              maximumBet:
+                $ref: '#/definitions/maximumBet'
+            required:
+              - playerIds
+              - minimumBet
+              - maximumBet
+            additionalProperties: false
+        required: 
+          - type
+          - attributes
+        additionalProperties: false
+    required:
+      - data
+    additionalProperties: false
+  gameResource:
+    properties:
+      id:
+        $ref: '#/definitions/gameId'
+      type:
+        $ref: '#/definitions/gameType'
+      attributes:
+        type: object
+        properties:
+          round:
+            $ref: '#/definitions/round'
+          playerIds:
+            $ref: '#/definitions/playerIds'
+          minimumBet:
+            $ref: '#/definitions/minimumBet'
+          maximumBet:
+            $ref: '#/definitions/maximumBet'
+          betPool:
+            $ref: '#/definitions/betPool'
+          tableCards:
+            $ref: '#/definitions/cards'
+      links:
+        $ref: '#/definitions/selfLink'
+  gameResult:
+    properties:
+      data:
+        $ref: '#/definitions/gameResource'
+  gameResults:
+    properties:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/gameResource"
+  gameType:
+    type: string
+    enum:
+      - game
+  maximumBet:
+    type: integer
+    minimum: 1
+    description: The maximum bet every player can place each round.
+    example: 2000
+  minimumBet:
+    type: integer
+    minimum: 1
+    description: The minimum bet every player have to place at least each round.
+    example: 1000
+  playerBet:
+    type: integer
+    minimum: 0
+    description: The bet from a specific player.
+  playerEmail:
+    type: string
+    example: abc@efg.com
+    format: email
+  memberExpAboveLevel:
+    type: integer
+    minimum: 0
+    description: Player's experience point above current level.
+    example: 1234
+  playerId:
+    type: string
+    example: '101'
+  playerIds:
+    type: array
+    items:
+      $ref: '#/definitions/playerId'
+  playerInGamePatchBody:
+    properties:
+      data:
+        type: object
+        properties:
+          id:
+            $ref: '#/definitions/playerId'
+          type:
+            $ref: '#/definitions/playerType'
+          attributes:
+            type: object
+            properties:
+              playerBet:
+                $ref: '#/definitions/playerBet'
+              playerStatus:
+                $ref: '#/definitions/playerStatus'
+              playerCards:
+                $ref: '#/definitions/cards'
+            additionalProperties: false
+            required:
+              - id
+              - type
```

Comment:
`required` and its items need to be unindented so that it's under the `data` object